### DISCUSSION TITLE
关于行内文本高亮的问题

### DISCUSSION THREAD

**Discussion by Brannua** [YunYouJun/hexo-theme-starry#166] [Q&A 疑难杂症]
大佬您好，请问主题是否有md这种高亮的语法：==高亮内容==

下面是我 typora 设置的截图

![demo](https://user-images.githubusercontent.com/38487617/132931850-9ff764e5-8d51-436c-aa53-5508264b40c0.png)

**Comment by YunYouJun** [comment]
高亮？这个应该是扩展语法，扩展语法其实和主题没关系，主要是替换 hexo markdown 解析器。

**Reply by Brannua** [reply]
大佬哪个解析器带这个语法呀

**Reply by YunYouJun** [reply]
我没有尝试过，也许可以考虑直接使用 HTML 标签。

> 或者 https://github.com/hexojs/hexo-renderer-markdown-it 也许有，但我不确定。HTML 应该总是可行的。

**Reply by Brannua** [reply]
好嘞，谢谢大佬！
